web/docs/tql2/operators.md

```diff
@@ -0,0 +1,152 @@
+# Operators
+
+import './operators.css';
+
+## Basic
+
+Operator | Description | Example
+---------|-------------|--------
+[`drop`](operators/drop) | Removes columns from the event | `drop name, metadata.id`
+[`head`](operators/head) | Keep only the first N events | `head 20`
+[`select`](operators/select) | Selects values and discards the rest | `select name, id=metadata.id`
+[`set`](operators/set) | Explicitly assigns one or multiple values | `set name="Tenzir", id=metadata.id`
+[`sort`](operators/sort) | Sorts the events by one or more expressions | `sort name, -abs(transaction)`
+[`summarize`](operators/summarize) | Aggregates events with implicit grouping | `summarize name, sum(transaction)`
+[`tail`](operators/tail) | Keep only the last N events | `tail 20`
+[`taste`](operators/taste) | Keep only N events of each type | `taste 1`
+[`where`](operators/where) | Keeps only events matching the predicate| `where name.starts_with("John")`
+
+## Flow Control
+
+Operator | Description | Example
+---------|-------------|--------
+[`every`](operators/every) | Restarts a pipeline periodically | `every 10s { summarize sum(transaction) }`
+[`fork`](operators/fork) | Forwards a copy of the events to another pipeline | `fork { to "copy.json" }`
+[`group`](operators/group) | Starts a new pipeline for each group | `group path { to $path }`
+[`if`](operators/if) | Splits the flow based on a predicate | `if transaction > 0 { ... } else { ... }`
+[`match`](operators/match) | Splits the flow with pattern matching | `match name { "Tenzir" => {...}, _ => {...} }`
+[`timeout`](operators/timeout) | Ends a pipeline after a period without input | `timeout 1s { ... }`
+
+## Input
+
+Operator | Description | Example
+---------|-------------|--------
+[`from`](operators/from) | |
+[`load`](operators/load) | |
+[`load_file`](operators/load_file) | |
+[`load_http`](operators/load_http) | |
+
+
+Operator | Description | Example
+---------|-------------|--------
+**Fundamental** | *Basic operations expected everywhere*
```

Comment:
I could see 

1. a **Shaping** containing `set`, `enumerate`, and friends
2. a **Subsetting** (or **Filtering**) section for everything except `summarize` and `sort`, which could go into a **Analytics** section.